tests/cross_functional/system_test/test_clone_deletion_after_max_cluster_space_utilization.py

```diff
@@ -132,42 +130,55 @@ def test_clone_deletion_after_max_cluster_space_utilization(
         )
         clones_list = []
 
-        for clone_num in range(self.num_of_clones + 2):
-            logger.info(f"Start creation of clone number {clone_num}.")
-            cloned_pvc_obj = pvc_clone_factory(
-                self.pvc_obj, storageclass=self.pvc_obj.backed_sc, timeout=600
+        # Function to create clones
+        def create_clones(num_of_clones, start_num=0):
+            for clone_num in range(start_num, start_num + num_of_clones):
+                logger.info(f"Start creation of clone number {clone_num}.")
+                cloned_pvc_obj = pvc_clone_factory(
+                    self.pvc_obj, storageclass=self.pvc_obj.backed_sc, timeout=900
+                )
+                cloned_pvc_obj.reload()
+
+                if interface_type == constants.CEPHBLOCKPOOL:
+                    # flatten the image
+                    flatten_image(cloned_pvc_obj)
+                    logger.info(
+                        f"Clone with name {cloned_pvc_obj.name} of size {self.pvc_size}Gi was created."
+                    )
+                    cloned_pvc_obj.reload()
+                else:
+                    logger.info(
+                        f"Clone with name {cloned_pvc_obj.name} of size {self.pvc_size}Gi was created."
+                    )
+                clones_list.append(cloned_pvc_obj)
+
+        # Create the initial set of clones
+        create_clones(self.num_of_clones)
+
+        # Verify if expected alerts are seen; if not, continue creating extra clones
+        while True:
```

Comment:
using while with true is a dangerous operation as it can run indefinitely if the code block below does not reach the desired or expected state. Please avoid it

Reply:
Sure